src/pages/questions/ProductInformation.jsx

```diff
@@ -123,18 +137,67 @@ const ProductInformation = () => {
         labelPlacement="end"
       />
       {!hideImages && productData?.images && (
-        <Grid container rowSpacing={1.5} spacing={1}>
+        <Box
+          sx={{
+            display: "grid",
+            gridColumnGap: "100px",
+            gridRowGap: "30px",
+            gridTemplateColumns: "repeat(auto-fill, minmax(13rem, 1fr))",
```

Comment:
You know images will not take more than 300px
```suggestion
            gridTemplateColumns: "repeat(auto-fill, minmax(300px, 1fr))",
```

Reply:
Yes, I have rectified it.